trap.go

```diff
@@ -134,6 +133,13 @@ func main() {
 func trapHandler(packet *g.SnmpPacket, addr *net.UDPAddr) {
 	// log.Printf("got trapdata from %s\n", addr.IP)
 
+	if c.TrapServer.Community != packet.Community {
+		if c.Debug {
+			log.Printf("community mis match : %q %q", c.TrapServer.Community, packet.Community)
```

Comment:
packet.Communityでわかるのか、なるほどー!
"mismatch"かな、というのもありますが、機械解析するわけではないので人間向けのログとしてわかりやすくしてみる提案です

```suggestion
			log.Printf("invalid community: expected '%q', but received '%q'", c.TrapServer.Community, packet.Community)
```